client/src/Services/fetchData.js

```diff
@@ -0,0 +1,21 @@
+export default async function fetchData() {
+  try {
+    let allData = [];
+
+    const dataUrls = [
+      'http://localhost:4000/api/scores',
+      'http://localhost:4000/api/germannouns',
+    ];
+
+    const arrayOfPromises = dataUrls.map((url) =>
+      fetch(url)
+    );
+    for await (let request of arrayOfPromises) {
+      const data = await request.json();
+      allData.push(data);
+    }
+    return allData;
+  } catch (error) {
+    console.log(error.message);
```

Comment:
At some point I would recommend a visual error handling, so the user knows why she/he can't play 😅